### DISCUSSION TITLE
Means to disable auto correct when creating issues

### DISCUSSION THREAD

**Discussion by Deadlyelder** [community/community#8052] [Mobile]
Hello,

Truly enjoy having the Github Mobile app! Its a great welcomed change over emails about issues or PR for sure. Been using it quite a lot and the features are outstanding. 

Something that is missing and would actually make life quite simpler would be to not have auto correction or spell check within the app. It tends to be quite annoying when replying within PR or to an issue and suddenly there are auto capitalization or even auto correct of word when its not required. Would be awesome to have feature to turn off the auto correct and spell check for the Github.

Regards.

**Comment by brianlovin** [comment]
Thanks for the suggestion @Deadlyelder — I'm going to mention this to our team to see what's technically possible!
